bioptim/dynamics/configure_problem.py

```diff
@@ -804,6 +825,24 @@ def configure_taudot(nlp, as_states: bool, as_controls: bool):
         ConfigureProblem._adjust_mapping("taudot", ["qdot", "tau"], nlp)
         ConfigureProblem.configure_new_variable("taudot", name_taudot, nlp, as_states, as_controls)
 
+    @staticmethod
+    def configure_contact_forces(nlp, as_states: bool, as_controls: bool):
+        """
+        Configure the generalized forces derivative
+
+        Parameters
+        ----------
+        nlp: NonLinearProgram
+            A reference to the phase
+        as_states: bool
+            If the generalized force derivatives should be a state
+        as_controls: bool
+            If the generalized force derivatives should be a control
+        """
+
+        name_contact_forces = [f"{nlp.model.contactNames()[ii].to_string()}" for ii in range(nlp.model.nbContacts())]
```

Comment:
This can be reduced to:
`[name.to_string() for name in nlp.model.contactNames()]`

Reply:
done.

Reply:
done.